library/swift/src/ResponseHeaders.swift

```diff
@@ -4,6 +4,7 @@ import Foundation
 @objcMembers
 public final class ResponseHeaders: Headers {
   /// HTTP status code received with the response.
+  /// TODO: in Kotlin this blows up if the value of status cannot be translated to an int.
```

Comment:
@rebello95 Same here, do we error out when a `:status` is not a number?

Reply:
Commented above, but swift returns `nil` in the case of an invalid `:status` which I think makes sense, especially if at some point we have internal headers generated by Envoy/Envoy Mobile in the future that don't contain this header

Reply:
update kotlin with an optional. Although the server-side bias in me wishes we blew up.

Reply:
I feel we should blow up but let's keep it consistent for now and we can have a separate discussion about these details after. What do you think?